order-book/src/main/java/com/market/banica/order/book/model/ItemMarket.java

```diff
@@ -37,22 +54,116 @@ public void removeUntrackedItem(String itemName) {
         allItems.remove(itemName);
     }
 
+    public void updateItem(Aurora.AuroraResponse response) {
+        if (response.getMessage().is(TickResponse.class)) {
```

Comment:
please refactor so, that we throw an exception here, and remove the else clause

Reply:
Done